web/gds-user-ui/src/components/RegistrationForm/InvalidFormPrompt.tsx

```diff
@@ -55,12 +44,10 @@ function InvalidFormPrompt({
               <Text>
                 <Trans>
                   If you continue, your changes will be lost. To save your changes, click Cancel and
-                  then click on the
+                  then click on the{' '}
+                {isNextStep ? <Text as="span" fontWeight={700} whiteSpace={'break-spaces'}>Save & Next</Text> : <Text as="span" fontWeight={700} whiteSpace={'break-spaces'}>Save & Previous</Text>} button.
```

Comment:
Since the previous attempts didn't work, I think that wrapping text in `<Text>` tags should help resolve the rendering issue in staging and production.